kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -194,6 +194,11 @@ public void deactivate()
   }
 
   @Override
+  public void beforeCheckpoint(long windowId)
```

Comment:
I see, I undid that change in my second commit. @PramodSSImmaneni can you fix the first commit so it is cleaner.